modules/prelude-rust.el

```diff
@@ -36,23 +36,52 @@
 ;; * cargo (Rust Package Manager)
 ;; * racer (Rust Completion Tool)
 ;; * rustfmt (Rust Tool for formatting code)
+;; * rls (Rust Language Server, if the prelude-lsp feature is enabled)
 
 (prelude-require-packages '(rust-mode
-                            racer
-                            flycheck-rust
                             cargo))
 
+(when (featurep 'prelude-lsp)
```

Comment:
I think you should use `if/else` here.

Reply:
Probably you can put everything in just one big `if/else`, so it's easier to follow.

Reply:
Changed to `if/else`.